docs/conf.py

```diff
@@ -178,3 +178,55 @@
 
     edit_on_github_source_root = ""
     edit_on_github_doc_root = "docs"
+
+
+# Render rst pages as a jinja template to be able 
+# to generate rst docs from data (i.e. passed to template context) ---------
+def rstjinja(app, docname, source):
+    if app.builder.format != 'html':  # Make sure builder output is HTML
+        return
+    src = source[0]
+    context = app.config.html_context  # context dictionary, passed for rendering
+    context.update(facility_context(docname))
+    rendered = app.builder.templates.render_string(src, context)
+    source[0] = rendered
+
+def facility_context(docname):
+    dirsInPath = docname.split(os.sep)
```

Comment:
PEP8!

Reply:
Thanks for pointing out. 
Didn't catch my eye - it will take me time to get used to PEP8  :sweat_smile: